### PR TITLE
Support triple hyphen

### PR DESCRIPTION
**Pull request by Hypercubed**
Prettier configs are forcing `------------` to `---`.  With this change:

* Init now creates a CHANGELOG with a triple hyphen
* Subsequent actions preserve existing hyphen length

### CODE REVIEW

File: test/chg.test.js [hunk 1]
```diff
@@ -1,131 +1,138 @@
...
-    before(function() {
-      chg.delete();
-    });
-
-    it('deletes the current CHANGELOG.md file', function() {
-      expect(fs.existsSync('./CHANGELOG.md')).to.be.false;
-    });
-  });
-});
+'use strict';
```
Comment (heff) [inline test/chg.test.js]: What's going on with this giant diff?

Comment (Hypercubed) [reply test/chg.test.js]: hmmm... not sure... I did change things up a little so we can run tests in isolation.  Not sure why it looks like the whole file is new.

Comment (Hypercubed) [reply test/chg.test.js]: Maybe EOL I'm seeing inconsistent EOL across these files.

Comment (Hypercubed) [reply test/chg.test.js]: Yeah... for some reason a few files have DOS EOL chars... I can do another PR later that makes them all consistent.

**Comment by Hypercubed** [comment]
@heff All checks pass... what do you think?

**Pull request merged**